src/infiniteGrid.js

```diff
@@ -406,11 +406,14 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 		 * @return {eg.InfiniteGrid} instance of itself<ko>자신의 인스턴스</ko>
 		 */
 		setStatus: function(status) {
-			this.core.element.style.cssText = status.cssText;
-			this.core.$element.html(status.html);
+			if (!status || $.isEmptyObject(status)) {
```

Comment:
if all properties are needed, How about to change like below?
because `&&` in all lines make me difficult to read codes.

```javascript
if (!status || !status.cssText || !status.html || !status.core || !this.data) {
  return this;
}

//Same as before
this.core.element.style.cssText = status.cssText;
this.core.$element.html(status.html);
this.core.items = this.core.itemize(this.core.$element.children().toArray());
this.core.clone(this.core, status.core);
$.extend(this, status.data);
```